help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md

```diff
@@ -300,6 +300,7 @@ This section captures the functionality and significant documentation updates ma
 
 |Release month|Update type|Description|
 |---|---|---|
+|November 2023|Functionality update|<ul><li>We now automatically convert the **[UICONTROL Mapping ID]** values in the [schedule audience export](#schedule-segment-export-example) step to `lowercase`. Additionally we also sanitize `spaces` and `special characters`. This is done to ensure compliance with [!DNL Salesforce Marketing Cloud] field names.</li><li>We now support custom uploads, see the [supported audiences](#supported-audiences) section.</li></ul>|
```

Comment:
@mohan-h  - in addition to @vfalon 's comments on the other two PRs, please note that you need to replace: 

`This is done to ensure compliance with [!DNL Salesforce Marketing Cloud] field names` with Hubspot.

Reply:
Apologies, kindly re-review seems I mixed up the PR title and comments